### PR TITLE
Add `max-store-preparing-time` in pd-ctl

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a new config in pd-ctl

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): tikv/pd#4640

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -322,6 +322,17 @@ export PD_ADDR=http://127.0.0.1:2379 &&
     >> config set max-store-down-time 30m
     ```
 
+- `max-store-preparing-time` 为 store 上线阶段的最大等待时间，PD 可以查询处在上线阶段 store 的上线进度，当超过指定的时间后会被认为 store 已完成上线阶段，将无法再次查询这个节点的进度。
```
Comment (TomShawn) [inline pd-control.md:325]:
```suggestion
- `max-store-preparing-time` 控制 store 上线阶段的最长等待时间。在 store 的上线阶段，PD 可以查询该 store 的上线进度。当超过该配置项指定的时间后，PD 会认为该 store 已完成上线，将无法再次查询这个 store 的上线进度。
```

**Review by TomShawn — APPROVED** [review]
minor comment
rest LGTM

File: pd-control.md [hunk 2]
```diff
@@ -322,6 +322,17 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+
+    设置 store 上线阶段最多等待 4 小时：
+
+    {{< copyable "" >}}
+
+    ```bash
+    >> config set max-store-preparing-time 4h
+    ```
+
+
```
Comment (TomShawn) [inline pd-control.md:335]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**